core/swift57Action/CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# Apache OpenWhisk Swift 5.7 Runtime Container
+
+- Support for swift async/await
+
+## 1.17.0
```

Comment:
We'll fill in the actual number as part of making the release
```suggestion
## Next Apache Release
```

Reply:
https://github.com/apache/openwhisk-runtime-swift/pull/153/commits/4917c54a630c82b33a68ec0d66139b1657652ebe